website/addons/base/views.py

```diff
@@ -571,6 +571,7 @@ def addon_deleted_file(auth, node, error_type='BLAME_PROVIDER', **kwargs):
         'file_path': file_path,
         'file_name_title': file_name_title,
         'file_name_ext': file_name_ext,
+        'version_id': version.identifier,
```

Comment:
`version` isn't defined here. It might be `kwargs`, but even if so this will cause issues.